test/test_plugin_manager.rb

```diff
@@ -111,6 +111,7 @@ def with_no_gemfile
 
   context "plugins_dir is set to a different dir" do
     should "expand plugin path" do
+      skip "Jekyll doesn't support setting plugins_dir to a different dir on Windows"
```

Comment:
This message is misleading.  We would like to support it, it's because of Ruby bugs we can't.